security/azure/restrict_rdp_internet/azure_restrict_rdp_inet.pt

```diff
@@ -0,0 +1,203 @@
+name "Azure Restrict RDP from Internet"
+rs_pt_ver 20180301
+type "policy"
+short_description "A policy that sends email notifications when an Azure Network Security Group has RDP open to the internet. See the [README](https://github.com/flexera/policy_templates/tree/master/security/azure/restrict_rdp_internet) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+long_description ""
+severity "high"
+category "Security"
+default_frequency "hourly"
+info(
+  version: "2.0",
+  provider: "Azure",
+  service: "Network Security Group",
+  policy_set: "CIS",
+  cce_id: "",
+  benchmark_control: "6.1",
+  benchmark_version: "1.4.0",
+  cis_controls: "[\"9.2v7\"]",
```

Comment:
cis_controls should contain multiple entries. Specifically it should have, per the CIS document, 4.4v8, 4.5v8, and 9.2v7.

You can review other similar policies to see how to put more than one version number into CIS controls.